website/docs/kubernetes/creating-a-kube-cluster.md

```diff
@@ -0,0 +1,23 @@
+---
+sidebar_position: 11
+title: Creating a Kubernetes cluster
+description: Creating a Kubernetes cluster
+keywords: [podman desktop, podman, creating, cluster, kubernetes]
+tags: [creating-kubernetes-cluster, extentions]
+---
+
+# Creating a Kubernetes cluster using extensions
+
+Podman Desktop provides you the ability to set up a local Kubernetes cluster using different extensions. By installing these extensions, you can quickly configure a Kubernetes cluster with default or customized values. The following table covers the procedural sections for setting up a Kubernetes cluster:
```

Comment:
I feel like this sentence can be shorted to something like "Podman Desktop provides extensions such as Kind or MiniKube to start a local Kubernetes devevelopment cluster" or something similar?